adb/adb_commands.py

```diff
@@ -129,8 +130,30 @@ def Install(self, apk_path, destination_dir='', timeout_ms=None):
     basename = os.path.basename(apk_path)
     destination_path = destination_dir + basename
     self.Push(apk_path, destination_path, timeout_ms=timeout_ms)
-    return self.Shell('pm install -r "%s"' % destination_path,
-                      timeout_ms=timeout_ms)
+
+    cmds = ['pm install']
```

Comment:
call this `cmd` since it's only one command. Or `args` or `pieces` if you want it plural